SPECS/kubernetes/kubernetes.spec

```diff
@@ -268,6 +268,9 @@ fi
 %{_exec_prefix}/local/bin/pause
 
 %changelog
+* Mon May 06 2024 Rachel Menge <rachelmenge@microsoft.com> - 1.28.4-8
```

Comment:
This is release 7 in fast-track - please re-arrange and put `main`-only changes on top of it. It's OK to change dates for `main`-only changes.